core/templates/pages/preferences-page/preferences-page.component.html

```diff
@@ -232,13 +232,10 @@
     <hr ng-if="$ctrl.userCanDeleteAccount">
 
     <div ng-if="$ctrl.userCanDeleteAccount" role="form" class="form-group row">
-      <label class="col-lg-4 col-md-4 col-sm-4">Delete Account</label>
+      <label class="col-lg-4 col-md-4 col-sm-4" translate="I18N_DELETE_ACCOUNT_PAGE_HEADING"></label>
       <div class="col-lg-8 col-md-8 col-sm-8">
         <a href="/delete-account">
-          <button type="button"
-                  class="btn btn-danger protractor-test-delete-account-button"
-                  title="Delete account">
-            Delete account
+          <button type="button" class="btn btn-danger protractor-test-delete-account-button" title="Delete account" translate="I18N_DELETE_ACCOUNT_PAGE_HEADING">
           </button>
         </a>
         <span class="form-text oppia-form-text">This will take you to a page where you can delete your Oppia account.</span>
```

Comment:
Should we add a translation for this too?

Reply:
We can, actually. We should start translations for all of the text on the project, slowly.

Reply:
Do you want me to add it to this PR?

Reply:
Yes, I think it would make sense to include in this PR.

Reply:
I can add the text for English. We need to open a new issue for translations in other languages.

Reply:
Sure, no need for opening issues translation are handled by translation wiki.

Reply:
Btw, you will need to add these new fields to qqq.json, this file explains the purpose of the field so that translators can understand it.

Reply:
Added the stuff as you requested @vojtechjelinek. Someone needs to update the tests too to accommodate the new addition, I guess.